conan/tools/gnu/autotoolstoolchain.py

```diff
@@ -310,7 +310,8 @@ def _get_argument(argument_name, cppinfo_name):
                                        _get_argument("libdir", "libdirs"),
                                        _get_argument("includedir", "includedirs"),
                                        _get_argument("oldincludedir", "includedirs"),
-                                       _get_argument("datarootdir", "resdirs")])
+                                       _get_argument("datarootdir", "resdirs"),
+                                       _get_argument("libexecdir", "bindirs")])
```

Comment:
In the Conan Center Index repo, there are a couple of recipes using `bindirs` as the default value and mentioning this issue, but I just reviewed the GNU documentation, and it normally points to `{prefix}/libexec` instead.

I'd like to discuss it before merging this PR @memsharded @jcar87

Reply:
but Conan doesn't model "libexec", so the Conan info would be the ``bindirs``? At least for Windows, DLLs are in ``bindirs``, but in Linux the ``.so`` are in ``libdirs``